src/scss/base/_normalize.scss

```diff
@@ -0,0 +1,198 @@
+@mixin normalize() {
+    /**
+    * Normalize Styles
+    * =================================
+    * We use normalize.css for the bulk of our 'reset' (https://www.npmjs.com/package/normalize-scss).
+    * We wrap it into a mixin to be able to make it optional.
+    *
+    * This is an optional component within Fozzie.
+    * If you'd like to use it in your project you can include it by adding `@include normalize();` into your SCSS dependencies file.
+    */
+
+    /*! normalize-scss | MIT/GPLv2 License | bit.ly/normalize-scss v7.0.1*/
+    html {
```

Comment:
Just a QQ – how did you generate this file? Looking at the project, I can see the 3 SCSS files, so just wondering if when new versions come out and we want to update this.

Reply:
@ashleynolan I copied the output we have in the components.

Reply:
It might be worth following the NPM docs to use the approach stated on the package (can be done as a separate Engineering excellence ticket – just otherwise I'm not sure how we upgrade this if the normalise package upgrades in the future, as we won't be compiling it anywhere from the SCSS files then):

> Approach 1: Download and use normalize-scss as a starting point for your own project's base Sass, customising the values to match the design's requirements. (The best approach, IMO.)
> 
> Copy the normalize-scss files to your sass directory so that you can alter it as you include it in your project. To aid with this method, normalize-scss includes several ready-made "fork" versions: * fork-versions/default - Fork for libSass or Ruby Sass * fork-versions/typey - Fork with Typey * fork-versions/typey-chroma-kss - Fork with Typey, Chroma and KSS

Reply:
One other thing to consider is that the normalize-scss package appears to be using a very old version of normalize.css ([v5](https://github.com/JohnAlbin/normalize-scss#normalizecss-v5) compared to the [most recent v8](https://github.com/necolas/normalize.css)) and hasn't been updated for 4 years.

So maybe referencing `normalize-scss` is not the best approach?

Reply:
v8 is actually almost 3 years old itself — I'm guessing there aren't many reset updates to be made these days?

Reply:
Yeh, that's true. I guess we could just look to take in v8 itself then, as agree that it's not necessarily something that updates often. If we create a ticket for that, happy to merge this as-is

Reply:
I will create a ticket for this

Reply:
There is [a discussion](https://github.com/necolas/normalize.css/issues/856) in the normalize project about better maintained up-to-date alternatives. I will create a ticket to look into it and replace